lib/Event/ConfigureOptionsEvent.php

```diff
@@ -15,13 +15,22 @@
 
 class ConfigureOptionsEvent extends AbstractEvent
 {
+    /**
+     * @var OptionsResolverInterface
+     */
     private $options;
 
+    /**
+     * @param OptionsResolverInterface $options
+     */
     public function __construct(OptionsResolverInterface $options)
```

Comment:
@dantleech ... `OptionsResolverInterface` is deprecated since Symfony 2.6 and will be removed 3.0. Not sure why, but should we not use the `OptionsResolver` as suggested?